package.json

```diff
@@ -18,6 +18,9 @@
     "sql escape"
   ],
   "repository": "mysqljs/sqlstring",
+  "dependencies": {
+    "template-tag-common": "2.0.1"
```

Comment:
Bumped the major version because I reworked it to allow efficiently threading options objects through to address my failure to thread `timeZone` and `stringifyObjects` properly.